### PR TITLE
Fix a  typo in Tensorboard.tsx. Add a default TB version.

### PR DESCRIPTION
**Pull request by jingzhang36**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2832)

### CODE REVIEW

File: frontend/src/components/viewers/Tensorboard.tsx [hunk 1]
```diff
@@ -64,6 +64,9 @@ interface TensorboardViewerState {
   tensorflowVersion: string;
 }
 
+// TODO(jingzhang36): we'll later parse Tensorboard version from mlpipeline-ui-metadata.json file.
+const DefaultTensorboardVersion = '2.0.0';
```
Comment (Bobgy) [inline frontend/src/components/viewers/Tensorboard.tsx]: Convention is to name constants as `const DEFAULT_TENSORBOARD_VERSION = '2.0.0';`

Comment (jingzhang36) [reply frontend/src/components/viewers/Tensorboard.tsx]: done

**Pull request merged**